src/utils/daos_hdlr.c

```diff
@@ -1931,8 +1932,8 @@ file_opendir(struct file_dfs *file_dfs, const char *dir)
 static struct dirent*
 readdir_dfs(struct file_dfs *file_dfs, DIR *_dirp)
 {
-	int rc;
-	struct fs_copy_dirent *dirp = (struct fs_copy_dirent *)_dirp;
+	int	rc = 0;
+	struct	fs_copy_dirent *dirp = (struct fs_copy_dirent *)_dirp;
```

Comment:
This function is static, why is there a need to cast one of the input variables?

Reply:
these dfs wrappers are wrapped by file_ functions that call either the POSIX IO function or the DAOS IO function. The file_ function takes in a DIR *dirp, so these wrappers have similar arguments across POSIX, DAOS. Otherwise, fs_copy_dirent would have to be passed in even on calls for POSIX directories. For what its worth, these wrappers are going to be moved into DFS, and refactored there by @daltonbohning.